src/test/java/tests/PrintWorkoutsTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.DashboardPage;
+import pages.base.BaseTest;
+import pages.PrintWorkoutsPage;
+import utils.PropertyReader;
+
+public class PrintWorkoutsTest extends BaseTest {
+
+    public PrintWorkoutsPage printWorkoutsPage;
+
+    @BeforeMethod(alwaysRun = true)
+    public void loginAndMoveToPrintWorkouts() {
+        DashboardPage dashboardPage = loginPage.login(PropertyReader.getProperty("email"), PropertyReader.getProperty("password"));
+        printWorkoutsPage = dashboardPage.clickPrintWorkouts();
+    }
+
+    @Test(groups = "negative")
+    public void verifyPrintWithoutData() {
+        String expectedErrorMsg = """
+                ×
+                Please fix the following errors:
+                *The date range you have selected does not contain any workouts. Please close this browser window and adjust your date range.""";
+        printWorkoutsPage.print("07/07/2024", "07/09/2024");
+        Assert.assertEquals(printWorkoutsPage.getErrorText(), expectedErrorMsg,
+                "Error message does NOT appear");
+    }
+
+    @Test(groups = "positive")
+    public void verifyPrintWithData() {
+        printWorkoutsPage.print("07/12/2024", "07/12/2024");
```

Comment:
Are you sure that there always will be Workout data to print on these dates? What If I use my own account to run this test? - it will fail. So please add a pre-condition to this test - create a workout on selected date